metadata/stream/rationale.yaml

```diff
@@ -22,7 +22,7 @@ architectures:
               location: https://artifacts.example.com/dsB2fnzP7KhqzQ5a.vmdk.xz
               signature: https://artifacts.example.com/dsB2fnzP7KhqzQ5a.vmdk.xz.sig
               sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
-              # Could also include artifact size/uncompressed-size/uncompressed-sha256 from meta.json
+              uncompressed-sha256: 38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662
```

Comment:
Can we also copy/paste this to the other artifacts so it's clear it's not just an AWS thing?

Reply:
@cgwalters Looks like this still needs doing?